<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Magento Web API Functional Tests" tests="18" assertions="0" errors="18" failures="0" skipped="0" time="10.445924">
    <testsuite name="Magento\Webpos\Api\Cart\CartTaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="6.310882">
      <testcase name="testGetList" class="Magento\Webpos\Api\Cart\CartTaxClassTest" classname="Magento.Webpos.Api.Cart.CartTaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php" line="24" assertions="0" time="6.310882">
        <error type="Exception">Magento\Webpos\Api\Cart\CartTaxClassTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 596990001591&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php:40
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Cart\CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="1.858380">
      <testcase name="testSendEmail" class="Magento\Webpos\Api\Cart\CartTest" classname="Magento.Webpos.Api.Cart.CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php" line="24" assertions="0" time="1.858380">
        <error type="Exception">Magento\Webpos\Api\Cart\CartTest::testSendEmail
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 293151100930&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php:36
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.147004">
      <testcase name="testGetList" class="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" classname="Magento.Webpos.Api.Catalog.SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" line="35" assertions="0" time="0.147004">
        <error type="Exception">Magento\Webpos\Api\Catalog\SwatchRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1284062895493&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php:51
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" tests="5" assertions="0" errors="5" failures="0" skipped="0" time="0.656895">
      <testcase name="testSaveCart" class="Magento\Webpos\Api\Checkout\CheckoutTest" classname="Magento.Webpos.Api.Checkout.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" line="283" assertions="0" time="0.126951">
        <error type="Exception">Magento\Webpos\Api\Checkout\CheckoutTest::testSaveCart
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 943580106516&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:284
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSaveQuoteData" class="Magento\Webpos\Api\Checkout\CheckoutTest" classname="Magento.Webpos.Api.Checkout.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" line="406" assertions="0" time="0.131380">
        <error type="Exception">Magento\Webpos\Api\Checkout\CheckoutTest::testSaveQuoteData
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 563399772417&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:378
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:408
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSaveShippingMethod" class="Magento\Webpos\Api\Checkout\CheckoutTest" classname="Magento.Webpos.Api.Checkout.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" line="526" assertions="0" time="0.126110">
        <error type="Exception">Magento\Webpos\Api\Checkout\CheckoutTest::testSaveShippingMethod
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 619892432613&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:505
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:528
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testPlaceOrder" class="Magento\Webpos\Api\Checkout\CheckoutTest" classname="Magento.Webpos.Api.Checkout.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" line="734" assertions="0" time="0.135175">
        <error type="Exception">Magento\Webpos\Api\Checkout\CheckoutTest::testPlaceOrder
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 435146165818&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:622
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:736
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testRemoveItem" class="Magento\Webpos\Api\Checkout\CheckoutTest" classname="Magento.Webpos.Api.Checkout.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php" line="942" assertions="0" time="0.137279">
        <error type="Exception">Magento\Webpos\Api\Checkout\CheckoutTest::testRemoveItem
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 439726651841&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:925
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CheckoutTest.php:944
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Config\ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.145741">
      <testcase name="testGetList" class="Magento\Webpos\Api\Config\ConfigRepositoryTest" classname="Magento.Webpos.Api.Config.ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" line="24" assertions="0" time="0.145741">
        <error type="Exception">Magento\Webpos\Api\Config\ConfigRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 532863199757&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Customer\CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.294119">
      <testcase name="testGetList" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="35" assertions="0" time="0.144745">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1473389650099&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:57
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSave" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="83" assertions="0" time="0.149374">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testSave
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 291624272760&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:103
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.136433">
      <testcase name="testGetStockItems" class="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" classname="Magento.Webpos.Api.Inventory.StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" line="24" assertions="0" time="0.136433">
        <error type="Exception">Magento\Webpos\Api\Inventory\StockItemRepositoryTest::testGetStockItems
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 636687548521&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php:47
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Payment\PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.154455">
      <testcase name="testGetList" class="Magento\Webpos\Api\Payment\PaymentRepositoryTest" classname="Magento.Webpos.Api.Payment.PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" line="35" assertions="0" time="0.154455">
        <error type="Exception">Magento\Webpos\Api\Payment\PaymentRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 296204758855&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Pos\PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.140963">
      <testcase name="testAssignStaff" class="Magento\Webpos\Api\Pos\PosRepositoryTest" classname="Magento.Webpos.Api.Pos.PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" line="42" assertions="0" time="0.140963">
        <error type="Exception">Magento\Webpos\Api\Pos\PosRepositoryTest::testAssignStaff
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 703868009608&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php:52
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Sales\OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.144971">
      <testcase name="testGetList" class="Magento\Webpos\Api\Sales\OrderRepositoryTest" classname="Magento.Webpos.Api.Sales.OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" line="35" assertions="0" time="0.144971">
        <error type="Exception">Magento\Webpos\Api\Sales\OrderRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1030609341843&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php:57
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Staff\LoginTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.153058">
      <testcase name="testStaffLogin" class="Magento\Webpos\Api\Staff\LoginTest" classname="Magento.Webpos.Api.Staff.LoginTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php" line="32" assertions="0" time="0.153058">
        <error type="Exception">Magento\Webpos\Api\Staff\LoginTest::testStaffLogin
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 903882563721&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Staff\StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.165877">
      <testcase name="testStaffLogin" class="Magento\Webpos\Api\Staff\StaffManagementTest" classname="Magento.Webpos.Api.Staff.StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" line="31" assertions="0" time="0.165877">
        <error type="Exception">Magento\Webpos\Api\Staff\StaffManagementTest::testStaffLogin
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1129853204936&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\TaxClass\TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.137146">
      <testcase name="testSearchCriteria" class="Magento\Webpos\Api\TaxClass\TaxClassTest" classname="Magento.Webpos.Api.TaxClass.TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php" line="34" assertions="0" time="0.137146">
        <error type="Exception">Magento\Webpos\Api\TaxClass\TaxClassTest::testSearchCriteria
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 609204633614&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php:36
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
